internal/grpc_dialer.go

```diff
@@ -98,6 +107,8 @@ func dial(params dialParameters) (*grpc.ClientConn, error) {
 	}
 
 	opts = append(opts, securityOptions...)
+	opts = append(opts, grpc.WithDefaultCallOptions(grpc.MaxCallSendMsgSize(maxPayloadSize)))
```

Comment:
again, no need to override the send limitation

Reply:
I don't have a strong opinion on this one, did it for consistency reasons.